manifests/node-maintenance-operator/v0.6.0/node-maintenance-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -126,7 +126,7 @@ spec:
                   value: ""
```

Comment:
The issue still remains, `value` and `valueFrom` are mutually exclusive.

Reply:
Correct! pushed a fix for that now.